docs/NETWORKING.md

```diff
@@ -0,0 +1,44 @@
+# Networking
+> This module encapsulates the implementation of the network requests, including parsing of results and relevant model objects.  
+
+## High level class diagram
+![Networking high level class diagram](images/networking.png)
+
+## Remote
+A `Remote` performs network requests. This is the core of this module.  
+
+The base implementation of `Remote`  is provided an implementation of the `Network` protocol, and enqueues pairs of `Request` and `Mapper`, executing that request on the provided `Network` and delegating the parsing of the response to the `Mapper` provided. 
+
+There is a subclass of `Remote` for each relevant concern. Each of these subclasses provide public methods for each of the networking operations relevant to that concern.
+
+These are the current subclass of `Remote`:
```

Comment:
I would consider putting a qualifier on this list mentioning that there are probably more `Remotes`. Something like "at the time of writing this document". Just thinking of the likelihood of us making sure these docs are up-to-date. 😄